java/app-encryption/src/it/resources/config.yaml

```diff
@@ -0,0 +1,5 @@
+metastoreType: "memory"
+metastoreJdbcUrl: ""
```

Comment:
add params for username, password, connectionPoolSize for more flexible configuration